packages/mobile/src/account/saga.ts

```diff
@@ -109,9 +109,15 @@ function* clearStoredAccountSaga({ account }: ClearStoredAccountAction) {
     yield call(removeAccountLocally, account)
     yield call(clearStoredMnemonic)
     yield call(ValoraAnalytics.reset)
-    yield call(firebaseSignOut, firebase.app())
     yield call(deleteNodeData)
 
+    // Ignore error if it was caused by Firebase.
+    try {
+      yield call(firebaseSignOut, firebase.app())
```

Comment:
If Firebase isn't initialized it fails. It failed for me on the test build, since I guess it's not enabled there, and just in case something happens in the production build I think it's better to have this check because if this fails the account is actually deleted but the user is stuck on the settings page so they need to kill the app and open it again to have it work.

Reply:
Oh yes, firebase init is under a config.
This looks good then, however we should only log the error `if (FIREBASE_ENABLED)` to prevent confusions as to what error is expected.